whist/server/util.py

```diff
@@ -0,0 +1,20 @@
+"""Multi Thread adapter.."""
+import asyncio
+
+
+# pylint: disable=too-few-public-methods
+class ThreadManager:
+    """
+    Handles multi threads calls.
+    """
+
+    @staticmethod
+    def run(func, args) -> None:
+        """
+        Runs an async function in a new thread.
+        :param func: to be called
+        :param args: to be parsed to the call.
+        :return: None
+        """
+        loop = asyncio.new_event_loop()
```

Comment:
why are you manually creating a new event loop instead of just running them directly via asyncio.run or asyncio.create_task?

Reply:
I can have a look into it. It was the first to come to my mind.